editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/preferences/initializer/StagingRepositoriesPreferenceInitializer.java

```diff
@@ -0,0 +1,64 @@
+/*******************************************************************************
+ * Copyright (c) 2016 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.fusesource.ide.projecttemplates.preferences.initializer;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.eclipse.core.runtime.preferences.AbstractPreferenceInitializer;
+import org.eclipse.jface.preference.IPreferenceStore;
+import org.fusesource.ide.projecttemplates.internal.ProjectTemplatesActivator;
+
+public class StagingRepositoriesPreferenceInitializer extends AbstractPreferenceInitializer {
+	
+	private static final String NAME_URL_SEPARATOR = ",";
+	private static final String REPO_SEPARATOR = ";";
+	private static final String ENABLE_STAGING_REPOSITORIES = "enableStagingRepositories";
+	static final String STAGING_REPOSITORIES = "stagingRepositories";
+	private static final String PRODUCT_STAGING_REPO_URI = "fuse-internal"+NAME_URL_SEPARATOR+"http://download.eng.brq.redhat.com/brewroot/repos/jb-fuse-6.2-build/latest/maven";

```

Comment:
Ok. If it was there before, I guess I'm good with it. This just bubbles it up visibility wise